src/foam/dao/Relationship.js

```diff
@@ -324,3 +320,57 @@ foam.LIB({
     }
   ]
 });
+
+foam.CLASS({
+  package: 'foam.dao',
+  name: 'RelationshipPropertyValue',
+  properties: [
```

Comment:
Documentation: Are `DAOs` "complete" DAOs of each type, or filtered according to the source object (`o` in `o.relationshipPropertyName`)?